### PR TITLE
Powah quest rework

### PR DESCRIPTION
**Pull request by MuteTiefling**
Quest and quest loot overhaul.

Updated Configs to remove worldgen. Uraninite and Dry Ice are both crafted. 
image

image

Adjusted generation rates. Bringing things down a fair bit. 
Adjusted fuels, heat sources, and coolants. Coolants is on hold for now due to a bug making them work in reverse of expectation.

Starter Tier removed entirely. Basic Tier now uses a bit less Gold/Iron, instead using Thermal Series Alloys in places.
image
image
image
image

Tiny and basic capacitors removed as well. Only 'large' remains and has been renamed accordingly. Higher tier all use this as their base. 
image

Energizing recipes similarly modified to escape from vanilla material sink, instead sending the player off to Blue Skies for higher tier materials. 
image
image

Extra recipes for AE2:
Add one-step inscriber recipes to servo press:
image

Add Processor creation to Energizing Orb
image

### CODE REVIEW

File: kubejs/server_scripts/base/recipes/powah/shaped.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    //             pattern: ['BCB', 'AAA'],
+    //             key: {
+    //                 A: crystal,
+    //                 B: capacitor,
+    //                 C: Ingredient.of(lower_tiers.map((item) => `powah:solar_panel_${item}`))
+    //             },
+    //             id: `${id_prefix}solar_panel_${tier}`
+    //         }
+    //     );
+    // });
```
Comment (NielsPilgaard) [inline kubejs/server_scripts/base/recipes/powah/shaped.js:201]: Could this be removed? (a lot of commented out code)

**Review by NielsPilgaard — COMMENTED** [review]
The majority of changes look nice, but the Uraninite seems extremely expensive for normal mode. 
The Energizer recipes for the tiered materials also look to be a better fit in expert mode? Gating most of Powah behind Blue Skies isn't very kitchen sink friendly :D

File: kubejs/server_scripts/base/recipes/powah/shaped.js (hunk 1 above)
Comment (MuteTiefling) [reply kubejs/server_scripts/base/recipes/powah/shaped.js]: Sure, thing.

**Comment by MuteTiefling** [comment]
That's a fair point for the Blue Skies materials. Was mostly just looking to make use of some of the other materials in the pack rather than straight vanilla stuff. 

As for Uraninite, I was looking to just reduce the total energy density of the stuff, but I think it needed a re-check after some of the other config changes, since this is basically 50% of the energy it produces. Will drop the crafting cost a fair bit here.

**Comment by NielsPilgaard** [comment]
Awesome, thank you 👍

**Comment by MuteTiefling** [comment]
Uraninite now takes 10 million each and generates around 145 million each.
image

Nicer yields for using Blue Skies materials and a touch of magic:
Blazing Crystals
image
Niotic Crystals
image
Spirited Crystals:
image
Nitro Crystals: 
image

**Review by NielsPilgaard — APPROVED** [review]
Perfect :D

**Pull request merged**